Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -31,6 +31,10 @@
 
 MAXIMUM_STRING_MATCHES = 100
 
+ALLOWABLE_HOMEBREW_REPOSITORY_LINKS = [
+  %r{#{HOMEBREW_LIBRARY}/Homebrew/os/(mac|linux)/pkgconfig},
```

Comment:
```suggestion
  %r{#{Regexp.escape(HOMEBREW_LIBRARY)}/Homebrew/os/(mac|linux)/pkgconfig},
```

Not sure if this is needed?

Reply:
Yeah, doesn't hurt.